### ISSUE TITLE
The logo in your picture does not appear in wireshark, but the network card is indeed found in WiFi Sniffer. Why is this?

### ISSUE THREAD

**Issue by MissingDirection** [opendroneid/wireshark-dissector#21]
![wireshark](https://github.com/user-attachments/assets/8f807b05-0ae4-4e4f-a20e-67ba5a88e4d2)
![wifi sniffer](https://github.com/user-attachments/assets/0c06d132-37c2-4399-8197-6e19390c3c58)

**Comment by gabrielcox** [comment]
I'm unsure if the display changes with different versions.  Have you tried clicking on the configuration gear next to WiFi to select a channel?  If it lets you select the channel, try sniffing the channel with expected remote id packets. Are you seeing anything?

**Comment by MissingDirection** [comment]
Yes, I have tried it, but it seems to have no effect. Later, I replaced the hardware and solved the problem. Thank you for your reply
